layout/MDErgo1-Default-0.kll

```diff
@@ -27,9 +30,9 @@ U"D" : U"D";
 U"S" : U"S";
 U"A" : U"A";
 U"TAB" : U"LCTRL";
-U"LALT" : U"LCK1";
-U"LCTRL" : U"LCTRL";
-U"FUNCTION2" : U"LGUI";
+U"LALT" : U"LALT";
+U"LCTRL" : U"LGUI";
```

Comment:
`LGUI` and `LALT` are more similar to what the [Kinesis Advantage](https://www.kinesis-ergo.com/wp-content/uploads/2013/06/advantage_layout_mac.pdf) has